bin/eslint-formatters-calypso/src/cli.js

```diff
@@ -0,0 +1,19 @@
+const eslinesFormatter = require( './formatters/lines-modified-and-specific-rules' );
+
+/*
+ * This function should *not* call process.exit() directly,
+ * It should only return exit codes. This allows other programs
+ * to use the function and still control when the program exits.
+ *
+ */
+
+module.exports = function( report ) {
+	const newReport = eslinesFormatter( report );
+	if ( newReport ) {
+		process.stdout.write( newReport );
+
+		return 1;
+	}
+    // it has nothing to show
```

Comment:
Right. `eslint` doesn't reported any indentation error, though. It seems to be a known issue https://github.com/eslint/eslint/issues/3845

More info, for the record:

I have made _unvisible characters_ (such as tabs, spaces and returns) to be "visible", and I see that the comment you mention is using spaces instead of tabs:
![indent_atom_with_invisible_characters](https://cloud.githubusercontent.com/assets/19974613/17242459/76db9ac2-5577-11e6-9fa2-9b494d9a279a.png)

I'll take a quick review of the source code to fix all those.